index.bs

```diff
@@ -176,6 +166,27 @@ run these steps:
 
 </div>
 
+A <dfn export>file system path</dfn> is an implementation-defined type
+representing a potential location of a [=/file system entry=]. For example, this
+may be to a virtual path that is mapped to real location on disk or in memory,
+may correspond directly to a path on the local file system, or may not
+correspond to any file on disk at all.
```

Comment:
I'm happy to take suggestions on how to improve this definition, but it needs to be broad enough to allow sites to store as much information about the path/locator as needed, such as the context the path exists in (e.g. its bucket - see https://github.com/whatwg/fs/pull/30#issuecomment-1139068595)

Reply:
To allow sites or to allow user agents?

I think stating this as "A file system path represents the potential location of a file system entry." might be sufficient. As long as we're not directly exposing it, we don't have to define the details.

As for "locate", would it always return a file system entry or would it sometimes return nothing at all? E.g., if you need to create it first? I would probably make that a separate algorithm that takes a file system path. It seems a bit weird for something that I imagine would mostly be represented as a string or byte sequence to hold a concept such as locate.

Reply:
> To allow sites or to allow user agents?

Apologies, I meant "user agents"

> I think stating this as "A file system path represents the potential location of a file system entry." might be sufficient. As long as we're not directly exposing it, we don't have to define the details.
> 
> As for "locate", would it always return a file system entry or would it sometimes return nothing at all? E.g., if you need to create it first? I would probably make that a separate algorithm that takes a file system path. It seems a bit weird for something that I imagine would mostly be represented as a string or byte sequence to hold a concept such as locate.

Ack. I'll change the `[file system path/locate]` method to a "locate" function that takes a path